docs/faq.rst

```diff
@@ -161,3 +161,40 @@ I am seeing an Exception error, what's wrong?
 
 Have a look at the :ref:`Troubleshooting section <troubleshooting>` to
 see what to do.
+
+If two users modify the same collection offline, how does that conflict get resolved?
+-------------------------------------------------------------------------------------
+
+It depends on which client you use. If you use kinto.js, for instance, you can set up
```

Comment:
We shouldn't talk about client here, all client can and should implement the three thing. You can even implement it at the HTTP level.

We should rephrase like that:

    There are three conflict resolution strategies: